README.md

```diff
@@ -2,35 +2,327 @@
+### Knowing Your Printers
+To enable accurate customization of settings, developers need to have knowledge about the capabilities of the printer. The proposed API provides a convenient method for accessing this information, which enhances the printing experience by allowing developers to retrieve relevant details about what the printer can and cannot do and apply this knowledge appropriately. A few examples:
+* If the printer supports high-resolution printing, the application can automatically select the appropriate print resolution, resulting in sharper and more detailed output.
+* Access to supported media sizes empowers the application to generate print-ready files side by adjusting page layouts, margins, or scaling, effectively ensuring that the content fits properly on the printed page without any cropping or distortion.
 
-### Leveraging Progressive Web Apps
-The new Web API can motivate developers to switch to Progressive Web Apps (PWAs) by offering enhanced printing capabilities that were traditionally limited to native applications (in particular, support for Printer Redirection).
+### Observing the Printing Pipeline & Improving Error Handling
+When using the `window.print()` function for web printing, the developers do not have any visibility into the printing process at all, whereas the users are limited to browser notifications and obscure system UI surfaces that are not always easy to discover. This can lead to a frustrating experience for users, for instance, if they encounter errors but do not receive any guidance on how to resolve them, or if the print job gets stuck in the queue for long.
+
+While these problems could be somewhat softened by improving the existing notification system and making it easier for users to understand the progress of print jobs, allowing the applications to monitor them directly could also be beneficial for the users in a number of ways.
+* With more control over error handling, applications can wrap up raw printing errors into more elaborate descriptions (what went wrong and why?), suggest useful tips for troubleshooting common errors (potentially even going as far as providing links to vendor websites or user manuals for specific printers) or even prompt to print anew after the problem has been fixed.
+* Granting the application visibility into the progress of print jobs would allow the apps to present this information in a more graphical & convenient manner.
+
+## Main Motivating Use Case: Remote Printing
+The incremental improvements to the existing printing stack discussed in the previous sections do not address the crucial case of remote printing which has specific requirements that do not fit into the general web document-based model.
```

Comment:
Can you define "remote printing" more explicitly as being part of a remote desktop slash virtual desktop system? The comment below says "in many cases" but in this scenario it is every case.